src/test/java/org/openstreetmap/atlas/checks/persistence/SparkFileHelperTest.java

```diff
@@ -0,0 +1,130 @@
+package org.openstreetmap.atlas.checks.persistence;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.streaming.resource.Resource;
+
+import com.google.common.base.Objects;
+import com.google.common.io.Files;
+
+/**
+ * Tests for {@link SparkFileHelper}.
+ *
+ * @author mkalender
+ */
+public class SparkFileHelperTest
+{
+    // A helper without any specific configuration
+    // Empty configuration targets local filesystem
+    private static final SparkFileHelper TEST_HELPER = new SparkFileHelper(Collections.emptyMap());
+
+    @Test(expected = CoreException.class)
+    public void testDeleteNonExistingDirectory()
+    {
+        // Generate a temp path
+        final File tempFolder = Files.createTempDir();
+        final String path = tempFolder.getAbsolutePath();
+        tempFolder.delete();
+
+        // Try to delete directory
+        TEST_HELPER.deleteDirectory(path);
+    }
+
+    @Test
+    public void testIsDirectory() throws IOException
+    {
+        // Test a file
+        final File tempFile = File.createTempFile("test", ".tmp");
```

Comment:
Nit: You could use `FileSuffix.TEMPORARY` value instead of `".tmp"` String for these tests.